{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/.github/workflows/terraform-cd-prod.yml

```diff
@@ -0,0 +1,84 @@
+# This GitHub workflow deploys Terraform resources (ML resource config and more)
+# defined under {{cookiecutter.project_name}}/terraform, when PRs are merged into the default branch
```

Comment:
nit `when PRs are merged into the default branch` => `when PRs are merged into the release branch`